alacritty_terminal/src/config/mod.rs

```diff
@@ -280,51 +204,8 @@ impl FromString for Option<Shell<'_>> {
     }
 }
 
-/// A delta for a point in a 2 dimensional plane
-#[serde(default, bound(deserialize = "T: Deserialize<'de> + Default"))]
-#[derive(Clone, Copy, Debug, Default, Deserialize, PartialEq, Eq)]
-pub struct Delta<T: Default + PartialEq + Eq> {
-    /// Horizontal change
-    #[serde(deserialize_with = "failure_default")]
-    pub x: T,
-    /// Vertical change
-    #[serde(deserialize_with = "failure_default")]
-    pub y: T,
-}
-
-/// Wrapper around f32 that represents an alpha value between 0.0 and 1.0
-#[derive(Clone, Copy, Debug, PartialEq)]
-pub struct Alpha(f32);
-
-impl Alpha {
-    pub fn new(value: f32) -> Self {
-        Alpha(if value < 0.0 {
-            0.0
-        } else if value > 1.0 {
-            1.0
-        } else {
-            value
-        })
-    }
-}
-
-impl Default for Alpha {
-    fn default() -> Self {
-        Alpha(1.0)
-    }
-}
-
-impl<'a> Deserialize<'a> for Alpha {
-    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
-    where
-        D: Deserializer<'a>,
-    {
-        Ok(Alpha::new(f32::deserialize(deserializer)?))
-    }
-}
-
 #[derive(Deserialize, Copy, Clone, Debug, PartialEq, Eq)]
-struct DefaultTrueBool(bool);
+pub struct DefaultTrueBool(pub bool);
```

Comment:
Why is this made public? This shouldn't be used for anything other than deserialization.

Reply:
I'll just copy this to alacritty then, I was trying to avoid coping of this thing, but it's likely make sense to keep this private.